test/e2e/e2e_spec.yml

```diff
@@ -0,0 +1,39 @@
+description: |
+  End-to-end tests for Docker integration
+
+scenarios:
+  - description: |
+      This scenario will verify that metrics from docker integration are correcly collected.
+    before:
+      - docker-compose up -d
+    after:
+      - docker-compose down -v
+    integrations:
+      - name: nri-docker
+        when:
+          feature: docker_enabled
+          file_exists: /var/run/docker.sock
+        interval: 15s
+    tests:
+      nrqls:
+        - query: "SELECT average(docker.container.cpuUsedCores) FROM Metric"
+      entities:
+        - type: "CONTAINER"
+          data_type: "Metric"
+          metric_name: "docker.container.cpuUsedCores"
+      metrics:
+        - source: "docker.yml"
+          except_metrics:
+            - docker.container.memorySoftLimitBytes
+            - docker.container.pids
```

Comment:
what's the problem with pids?

Reply:
@marcsanmi 
From the samples I've seen `pids` are not frequently reported. As such I decided to exclude them from the list to avoid test failure. They don't seem to be a standard/basic metric

Reply:
It was out of curiosity, but seems we have never reported them... So better skip them 👍